### ISSUE TITLE
Code Review Protocol

### ISSUE THREAD

**Issue by Wmaxlees** [acm-ucdenver/website#106]
As a developer, I would like a clear protocol for how code reviews are processed.

**Comment by robfitzgerald** [comment]
some thoughts.

at work, we created a slack channel for "git-pull" where pull requests would be announced.  we made a rule that 2 team members needed to :+1: a pull request before it would be merged, the second one doing the merging assuming it can be 'automatically merged with no conflicts'.

i like that we both look at the pull requests, for now.  maybe once people feel confident about their coding techniques, we could make a more egalitarian review like the one at my work.  in the meantime, we could just both have to give it our plus one, and write comments on the pull request itself if there are any changes needed before merging.

**Comment by robfitzgerald** [comment]
i guess we don't need the channel, since we are both using the github site w/ zenhub here.  at work some of us aren't using the github site and we communicate a lot in slack.  but, i would be happy just keeping an eye on pull requests via #github channel and github > Pull requests page and adding my :+1: and voice there.

**Comment by Wmaxlees** [comment]
Yeah, I think we should keep using the git hub pull request page. It's nice that you can make comments to specific lines of code. And if we have two pull requests at once, having a single channel to try and discuss both of them could get messy and comments get lost. I like the idea of needing 2 people to :+1: though.
